tests/test_git/test_git_init.py

```diff
@@ -0,0 +1,47 @@
+import sublime
+import os
+from .common import GSCreateRepoTestCase
+from GitSavvy.core import git_command
+
+
+def tidy_path(path):
+    return os.path.realpath(os.path.normcase(path))
+
+
+class TestInit(GSCreateRepoTestCase, git_command.GitCommand):
+
+    def test_01_window_path(self):
+        self.assertEqual(
+            tidy_path(self._temp_dir),
+            tidy_path(sublime.active_window().folders()[0]))
+        self.assertEqual(
+            tidy_path(self._temp_dir),
+            tidy_path(self.window.folders()[0]))
+
+    def test_02_add_file(self):
```

Comment:
A lot of git operations are sequential, I don't know if it is a better idea to put all the assertions in a single function than what I have now.